deployments/ecmps/release-v2.0-fix/Sprint8/6080/camdecmps/get-tee-feedback.sql

```diff
@@ -0,0 +1,34 @@
+DROP FUNCTION IF EXISTS camdecmps.get_test_extension_exemption_feedback(text) CASCADE;
+
+CREATE OR REPLACE FUNCTION camdecmps.get_test_extension_exemption_feedback(teeid text)
+RETURNS TABLE(
+    unit_stack_pipe character varying,
+    extens_exempt_cd character varying,
+    tee_year_quarter character varying,
+    sys_id_type text,
+    component text,
+    span_fuel character varying)
+LANGUAGE plpgsql
+AS $$
+BEGIN
+RETURN QUERY
+SELECT
+    COALESCE(ml.stack_name, ml.unitid) AS unit_stack_pipe,
+    tee.extens_exempt_cd,
+    rp.period_abbreviation AS tee_year_quarter,
+    ms.system_identifier || '/' || ms.sys_type_cd AS sys_id_type,
+    c.component_identifier || '/' || c.component_type_cd AS component,
+    COALESCE(tee.span_scale_cd, tee.fuel_cd) AS span_fuel
+FROM camdecmps.test_extension_exemption tee
+         INNER JOIN (
```

Comment:
See "deployments/ecmps/release-v2.0-fix/Sprint8/6080/camdecmps/get-qatest-feedback" comment on cl_filtered sub-query.

Reply:
See comment above...

Reply:
Thanks for the clarification and I agree with your goal.  I am going to resolve this conversation.